### PR TITLE
[frontend/backend] Default dashboard available in popover

### LINKED ISSUE #6843: Be able to select the default dashboard for home
## Use case

Be able to select the default dashboard for home.

### PR DESCRIPTION
**Pull request by Kedae**
Closes #6843

### CODE REVIEW

**Review by SarahBocognano — APPROVED** [review]

File: opencti-platform/opencti-front/src/private/components/DashboardSettings.jsx [hunk 1]
```diff
@@ -42,6 +42,8 @@ const useStyles = makeStyles(() => createStyles({
   }),
 }));
 
+export const DEFAULT_DASHBOARD = 'cf093b57-713f-404b-a210-a1c5c8cb3791';
```
Comment (SouadHadjiat) [inline opencti-platform/opencti-front/src/private/components/DashboardSettings.jsx]: what is this id ?

File: opencti-platform/opencti-front/lang/front/fr.json [hunk 2]
```diff
@@ -528,6 +528,7 @@
   "dashboard": "Tableau de bord",
   "Dashboard settings": "Paramètres du tableau de bord",
   "Dashboards": "Tableaux de bord",
+  "Dashboards from your groups & organizations": "Tableaux de bord de vos groupes & ; organisations",
```
Comment (SouadHadjiat) [inline opencti-platform/opencti-front/lang/front/fr.json]: why "& ;" in this translation?

Comment (Kedae) [reply opencti-platform/opencti-front/lang/front/fr.json]: Ho, didn't see that, deepl doesn't like & it seems :D I will update the translations

File: opencti-platform/opencti-front/src/private/components/DashboardSettings.jsx (hunk 1 above)
Comment (Kedae) [reply opencti-platform/opencti-front/src/private/components/DashboardSettings.jsx]: In order to be able to select the platform default dashboard within UI I added a hardcoded id representing this dashboard.

**Comment by SouadHadjiat** [comment]
What is the difference between "Platform dashboard" and "Default dashboard" ? I see both locally, and they display the same dashboard (the default one)

image

**Comment by Kedae** [comment]
> What is the difference between "Platform dashboard" and "Default dashboard" ? I see both locally, and they display the same dashboard (the default one)
> 
> image

As described in the Notion page : Default dashboard is the new name of "Automatic" that chooses the dashboard for you (either or platform or your first recommended dashboard).
Platform dashboard is the default dashboard provided by Filigran within OpenCTI at first launch.

**Comment by SouadHadjiat** [comment]
> > What is the difference between "Platform dashboard" and "Default dashboard" ? I see both locally, and they display the same dashboard (the default one)

> As described in the Notion page : Default dashboard is the new name of "Automatic" that chooses the dashboard for you (either or platform or your first recommended dashboard). Platform dashboard is the default dashboard provided by Filigran within OpenCTI at first launch.

Oh I thought it was the opposite : "default dashboard" being the default opencti dashboard, and "platform dashboard" the one selected by the admin on the group. It's a bit confusing in my opinion.

File: opencti-platform/opencti-front/src/private/components/DashboardSettings.jsx (hunk 1 above)
Comment (SouadHadjiat) [reply opencti-platform/opencti-front/src/private/components/DashboardSettings.jsx]: so this "DEFAULT_DASHBOARD" is the default of opencti ? but in the UI we named "default dashboard" the custom one selected for the group, and "platform dashboard" is the default. I think it's completely confusing and will be hard to remember that "default dashboard" in the ui doesn't mean "default_dashboard" in the code.

Comment (Kedae) [reply opencti-platform/opencti-front/src/private/components/DashboardSettings.jsx]: I renamed the constant

**Pull request merged**